drivers/at86rf2xx/at86rf2xx_netdev.c

```diff
@@ -211,18 +211,28 @@ static int _recv(netdev_t *netdev, void *buf, size_t len, void *info)
      */
     if (info != NULL) {
         uint8_t ed = 0;
+        uint8_t crc = 0;
```

Comment:
```suggestion
        uint8_t crc_valid = 0;
```

Just `crc` is a misleading name. I'd assume it would contain the value of the CRC.